packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -325,21 +323,39 @@ export default {
   }
 
   .oc-card-media-top {
-    display: inline-block;
     width: 100%;
-    background-color: var(--oc-color-background-muted);
-    height: 200px;
+    height: 150px;
   }
+
   .oc-card-media-top a {
     display: flex;
     justify-content: center;
     align-items: center;
     height: 100%;
   }
+
+  .oc-tag {
+    color: var(--oc-color-text-default);
+  }
+
   .space-image {
     width: 100%;
-    height: 200px;
+    height: 150px;
     object-fit: cover;
+    border-top-left-radius: 3px;
```

Comment:
You can use the css class `oc-rounded-top` here, which will set the top left and top right radius to 5px. FYI: We have helper classes for top, right, bottom, left and all around.